prowler/providers/azure/services/network/network_watcher_enabled/network_watcher_enabled.py

```diff
@@ -0,0 +1,29 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.network.network_client import network_client
+
+
+class network_watcher_enabled(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+        nw_locations = []
+        for subscription, network_watchers in network_client.network_watchers.items():
+            for network_watcher in network_watchers:
+                nw_locations.append(network_watcher.location)
+        for subscription, security_groups in network_client.security_groups.items():
+            for security_group in security_groups:
```

Comment:
Iterate with the locations that are in audit info, please

Reply:
Done